### PR TITLE
Python API - Specialized Pipes for SnowFlake, BigQuery etc.

### PR DESCRIPTION
**Pull request by navinrathore**
* setData(), setOutput() modified to handle multi-pipe configuration
* added python pipes for snowflake, inmemory, bigquery, csv

### CODE REVIEW

**Comment by navinrathore** [comment]
#322  (item no. 1, 3)

File: python/phases/zinggPipes.py [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    def setTemporaryGcsBucket(self, bucket):
+        Pipe.addProperty(self, "temporaryGcsBucket", bucket)
+    def setViewsEnabled(self, isEnabled):
+        Pipe.addProperty(self, "viewsEnabled", isEnabled)
+
+class SnowflakePipe(Pipe):
+    def __init__(self,name):
+        Pipe.__init__(self, name, Format.SNOWFLAKE.type())
+    def setURL(self, url):
+        Pipe.addProperty(self, "sfUrl", url)
```
Comment (sonalgoyal) [inline python/phases/zinggPipes.py:35]: Please refer to java code properties defined

File: python/phases/zinggPipes.py [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+        Pipe.addProperty(self, "sfUser", user)
+    def setPassword(self, passwd):
+        Pipe.addProperty(self, "sfPassword", passwd)
+    def setDatabase(self, db):
+        Pipe.addProperty(self, "sfDatabase", db)
+    def setSFSchema(self, schema):
+        Pipe.addProperty(self, "sfSchema", schema)
+    def setWarehouse(self, warehouse):
+        Pipe.addProperty(self, "sfWarehouse", warehouse)
+    def setDbtable(self, dbtable):
```
Comment (sonalgoyal) [inline python/phases/zinggPipes.py:46]: camel casing

File: python/phases/zingg.py [hunk 3]
```diff
@@ -158,6 +160,12 @@ def addProperty(self, name, value):
...
         self.pipe.setSchema(s)
 
+    def getDataset(self):
+        return self.pipe.getDataset()
+
+    def toString(self):
+        return self.pipe.toString()
+
 def parseArguments(argv):
     parser = argparse.ArgumentParser(description='Zingg\'s python APIs')
```
Comment (sonalgoyal) [inline python/phases/zingg.py:170]: Zingg Python API

File: python/phases/zinggPipes.py (hunk 1 above)
Comment (navinrathore) [reply python/phases/zinggPipes.py:35]: Done

File: python/phases/zinggPipes.py (hunk 2 above)
Comment (navinrathore) [reply python/phases/zinggPipes.py:46]: Done

File: python/phases/zingg.py (hunk 3 above)
Comment (navinrathore) [reply python/phases/zingg.py:170]: moved to InMemory

**Pull request merged**